### ISSUE TITLE
Add Muzzle Flashes

### ISSUE THREAD

**Issue by poosh** [NVIDIA/Q2RTX#276]
Firing a weapon looks ridiculous without a muzzle flash. I don't know why the original Q2 didn't have it (the N64 port has it, though), but it would be a neat add-on to the remaster.

**Comment by appetrosyan** [comment]
Original Q2 didn't have those either. While I get that Q2 RTX does a lot of things that the original didn't, I'm not sure that implementing it would be a great idea.

**Comment by poosh** [comment]
Yes, the original didn't have muzzle flashes, but even N64 port had them. Ray-traced lighting coming from flashes would look spectacular.

**Comment by apanteleev** [comment]
Lighting alone isn't enough for a spectacular look though, proper muzzle flashes need some fire effects. The effect needs to be modeled, textured, placed at the end of each applicable gun's muzzle, and synchronized with the light.

**Comment by QuakeMM** [comment]
https://github.com/NVIDIA/Q2RTX/assets/74773831/bbeb6663-7621-4cf6-99ba-0d5842205608

**Comment by apanteleev** [comment]
The project is discontinued; no new features or fixes will be implemented here. Closing.
